receiver/k8sclusterreceiver/e2e_test.go

```diff
@@ -100,6 +100,8 @@ func TestE2E(t *testing.T) {
 		pmetrictest.ChangeResourceAttributeValue("k8s.namespace.uid", replaceWithStar),
 		pmetrictest.ChangeResourceAttributeValue("k8s.daemonset.uid", replaceWithStar),
 		pmetrictest.ChangeResourceAttributeValue("container.image.name", containerImageShorten),
+		pmetrictest.IgnoreResourceSchemaURL(),
+		pmetrictest.IgnoreScopeVersion(),
```

Comment:
Do we need these given that you updated `e2e/expected.yaml`?

Reply:
we do. I get an error "latest" and "0.79-dev" clashing otherwise.

Reply:
That's for the scope version, but the resource schema URL is fixed, no?

Reply:
The schema url is only present on the metrics I moved over, and therefore when we sort resource metrics change the order, which breaks comparison.

Reply:
If all the schema url fields match (doesn't matter if it's zero value or not) the order should not be affected. Likely you just missed updating some of them.

Reply:
ah, whitespace issue. Looking.